### ISSUE TITLE
[Bug]: search bar ERROR: type List(Float64) is incompatible with expected type String 

### ISSUE THREAD

**Issue by devMls** [infiniflow/ragflow#3910]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### RAGFlow workspace code commit ID

dont now

### RAGFlow image version

dev

### Other environment information

```Markdown
infinity
openai 4o mini chat
embbeding and reranking voyage
```

### Actual behavior

When I try to search something in my new test (fresh install) I obtain a error:

![image](https://github.com/user-attachments/assets/e22c146b-8668-419c-b5c0-2725850c5f2d)

The trace:

```
2024-12-06 11:30:43,995 INFO     15 172.17.0.3 - - [06/Dec/2024 11:30:43] "POST /v1/chunk/retrieval_test HTTP/1.1" 200 -

2024-12-06 11:30:53,961 INFO     15 HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"

2024-12-06 11:30:53,965 INFO     15 172.17.0.3 - - [06/Dec/2024 11:30:53] "POST /v1/conversation/related_questions HTTP/1.1" 200 -

2024-12-06 11:30:54,954 ERROR    15 type List(Float64) is incompatible with expected type String

Traceback (most recent call last):

  File "/ragflow/api/apps/chunk_app.py", line 302, in retrieval_test

    ranks = retr.retrieval(question, embd_mdl, tenant_ids, kb_ids, page, size,

  File "/ragflow/rag/nlp/search.py", line 311, in retrieval

    sres = self.search(req, [index_name(tid) for tid in tenant_ids], kb_ids, embd_mdl, highlight)

  File "/ragflow/rag/nlp/search.py", line 105, in search

    res = self.dataStore.search(src, highlightFields, filters, matchExprs, orderBy, offset, limit, idx_names, kb_ids)

  File "/ragflow/rag/utils/infinity_conn.py", line 299, in search

    res = concat_dataframes(df_list, selectFields)

  File "/ragflow/rag/utils/infinity_conn.py", line 57, in concat_dataframes

    return pl.concat(df_list)

  File "/ragflow/.venv/lib/python3.10/site-packages/polars/functions/eager.py", line 185, in concat

    out = wrap_df(plr.concat_df(elems))

polars.exceptions.SchemaError: type List(Float64) is incompatible with expected type String
```

### Steps to reproduce

```Markdown
search something
```

**Comment by JinHai-CN** [comment]
@devMls Would you please check the version of RAGFlow? Check the 'System' page of user.

**Comment by devMls** [comment]
yes:

0d5486aa(v0.14.1~75) slim

**Comment by yuzhichang** [comment]
@devMls https://github.com/infiniflow/ragflow/pull/4059 should have fixed this, and will be released in 0.15.0.
